zizmor.yml

```diff
@@ -0,0 +1,4 @@
+rules:
+  cache-poisoning:
+    ignore:
+      - build-binary-preview.yml
```

Comment:
What is the potential cache poisoning attack that we're ignoring a warning about? Is it definitely a false positive?